projects/ccf-eui/src/app/app.component.ts

```diff
@@ -10,6 +10,7 @@ import { DataQueryState, DataState } from './core/store/data/data.state';
 import { FiltersPopoverComponent } from './modules/filters/filters-popover/filters-popover.component';
 import { ImageViewerPopoverComponent } from './modules/image-viewer/image-viewer-popover/image-viewer-popover.component';
 import { DrawerComponent } from './shared/components/drawer/drawer/drawer.component';
+import { ALL_ORGANS } from '../../../ccf-shared/src/lib/components/organ-selector/organ-selector.component';
```

Comment:
Don't use relative imports to import from ccf-shared. I'm suprised this actually worked :-O, should be something like import { ALL_ORGANS } from 'ccf-shared'; . Fix this wherever you've been importing from ccf-shared. You'll need to also add organ selector to the public-api.ts of ccf-shared.